oml/models/vit/clip.py

```diff
@@ -0,0 +1,159 @@
+from pathlib import Path
+from typing import Any, Dict, Optional
+
+import torch
+
+from oml.interfaces.models import IExtractor
+from oml.models.utils import (
+    filter_state_dict,
+    patch_device_and_float,
+    remove_prefix_from_state_dict,
+)
+from oml.models.vit.vision_transformer_clip import VisionTransformer
+from oml.utils.io import download_checkpoint
+
+
+class ViTCLIPExtractor(IExtractor):
+    pretrained_models: Dict[str, Any] = {
```

Comment:
can we squeeze the size of this dictionary like in ViTExtractor?
1. shorter the links moving prefix to the variable 
2. use only the first 6 letters of hash

thus, we may try to place everything in place